notesapi/v1/views.py

```diff
@@ -39,6 +39,8 @@ class AnnotationSearchView(GenericAPIView):
 
```

Comment:
I would re phrase it something like this:
` Search and return a list of annotations for a user.`